sagemaker-debugger/xgboost_training_report/higgs_boson_detection.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<h2>Setup</h2><p>This notebook was created and tested on an <code>ml.t3.medium</code> notebook instance.</p><p>After we've installed and imported the required packages, we'll need to specify a few variable that will be utilized throughout the example notebook:</p><ul><li><code>role</code>: The IAM role to run SageMaker training jobs. The default SageMaker role with the SageMaker full access policy will be used.</li><li><code>sess</code>: The SageMaker session that interacts with different AWS services.</li><li><code>bucket</code>: The S3 bucket where the model's input and output data will be stored. We will use the default S3 bucket automatically paired with the SageMaker session.</li><li><code>key_prefix</code>: The directory in the S3 bucket where we'll store the input and output data</li><li><code>region</code>: The AWS region where we operate the SageMaker training job.</li><li><code>s3</code>: the s3fs client to make it easier to read and write data from and to the S3 bucket.</li><li><code>xgboost_container</code>: The URI for the XGBoost training container for our region.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1947/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>